application/cms/data_utils.py

```diff
@@ -230,6 +230,7 @@ def __get_v2_table_values(table_settings):
                 if table_settings["tableOptions"]["table_column_5"] != "none"
                 else None
             ),
+            "table_first_column_name": table_settings["tableOptions"]["table_category_caption"],
```

Comment:
It feels confusing to have `table_column_1_name` and `table_first_column_name` being different things.  Sam doesn't like `column_0`, which I thought was perfectly reasonable :trollface: 

Something like "ethnicity_label_overrirde"?  Or "index_column_name" (if that makes any sense)?

Reply:
I'll call it "table_index_column_name"